lib/accesscontrol/permitted_action_query.rb

```diff
@@ -31,6 +31,28 @@ def can?(action_id, object_type)
       end
     end
 
+    # Allow permission on a general action in the given namespace represented by object_type.
+    # A grant is universally unique and is enforced at the database level.
+    #
+    # @param action_id [Integer] The action to grant for the object
+    # @param object_type [String] The namespace of the given action_id.
+    # @return [nil] Returns nil if successful, otherwise will raise an Error (AccessControl::CouldNotGrantError).
```

Comment:
Again, you can use yardoc's [@raise](http://www.rubydoc.info/gems/yard/file/docs/Tags.md#raise) syntax to describe raised exceptions.

Reply:
👍